### PR TITLE
TASK-39751: Manager url in a space

### PR DESCRIPTION
**Pull request by GouadriaHanen**

### CODE REVIEW

File: webapp/portlet/webpack.dev.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const webpackCommonConfig = require('./webpack.common.js');
 const app = 'social-portlet';
 
 // add the server path to your server location path
-const exoServerPath = "/exo-server";
+const exoServerPath = "/home/exo/work/exoRepos/plf-enterprise-tomcat-standalone-6.2.x-20210127.094402-162/platform-6.2.x-SNAPSHOT";
```
Comment (ahmadmido) [inline webapp/portlet/webpack.dev.js]: Restore this file please, it's shouldn't be changed

Comment (ahamdi) [inline webapp/portlet/webpack.dev.js]: restore the default configuration please.

Comment (GouadriaHanen) [reply webapp/portlet/webpack.dev.js]: already done !

File: webapp/portlet/src/main/webapp/space-infos-app/components/ExoSpaceInfos.vue [hunk 2]
```diff
@@ -20,7 +20,8 @@ export default {
   data() {
     return {
       description: '',
-      managers: []
+      managers: [],
+      url: `${eXo.env.portal.context}/${eXo.env.portal.portalName}/profile/`
```
Comment (ahamdi) [inline webapp/portlet/src/main/webapp/space-infos-app/components/ExoSpaceInfos.vue]:
```suggestion
      profileURL: `${eXo.env.portal.context}/${eXo.env.portal.portalName}/profile/`
```

File: webapp/portlet/src/main/webapp/space-infos-app/components/ExoSpaceInfos.vue [hunk 3]
```diff
@@ -6,7 +6,7 @@
       <h5>{{ $t("social.space.description.managers") }}</h5>
       <ul id="spaceManagers">
         <li v-for="manager in managers" :key="manager" class="spaceManagerEntry">
-          <a :href="manager.href">
+          <a :href="url+manager.username">
```
Comment (ahamdi) [inline webapp/portlet/src/main/webapp/space-infos-app/components/ExoSpaceInfos.vue]:
```suggestion
          <a :href="profileURL + manager.username">
```

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: webapp/portlet/webpack.dev.js (hunk 1 above)
Comment (GouadriaHanen) [reply webapp/portlet/webpack.dev.js]: restored!

File: webapp/portlet/src/main/webapp/space-infos-app/components/ExoSpaceInfos.vue (hunk 2 above)
Comment (aelbenney) [inline webapp/portlet/src/main/webapp/space-infos-app/components/ExoSpaceInfos.vue]:
```suggestion
      profileUrl: `${eXo.env.portal.context}/${eXo.env.portal.portalName}/profile/`
```

File: webapp/portlet/src/main/webapp/space-infos-app/components/ExoSpaceInfos.vue (hunk 3 above)
Comment (aelbenney) [inline webapp/portlet/src/main/webapp/space-infos-app/components/ExoSpaceInfos.vue]:
```suggestion
          <a :href="`${url}${manager.username}`">
```

**Review by aelbenney — CHANGES_REQUESTED** [review]

**Review by aelbenney — APPROVED** [review]

**Review by ahamdi — APPROVED** [review]

**Pull request merged**